README.md

```diff
@@ -93,6 +94,12 @@ sudo curl -sL github.com/justchokingaround/lobster/raw/main/lobster.sh -o /usr/l
 sudo chmod +x /usr/local/bin/lobster
 ```
 
+#### Nixos (Flake)
+
+```sh
+nix run github:justchokingaround/lobster#lobste
```

Comment:
missing r?

also this just runs the script, it doesnt install it, right?

Reply:
yes. It pulls the dependencies and runs it. Installing would be like this. I can add this to the readme.
```
environment.systemPackages = [
  inputs.lobster.packages.<architecture>.lobster
];
```

Reply:
yep, pls do, since it's an install section